pkgs/by-name/op/open-adventure/package.nix

```
@@ -0,0 +1,81 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  python3Packages,
+  pkg-config,
+  libedit,
+  cppcheck,
+  coreutils,
+  asciidoctor,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "open-adventure";
+  version = "1.20";
+  src = fetchFromGitLab {
+    owner = "esr";
+    repo = "open-adventure";
+    tag = finalAttrs.version;
+    hash = "sha256-xbsMz99CNLhpM6BSJVcRzxPB6tUYfPy/3Z+8BKt8b1E=";
+  };
+
+  nativeBuildInputs = [
+    python3Packages.python
+    pkg-config
+    asciidoctor
+  ];
+
+  buildInputs = [
+    python3Packages.pyyaml
+    libedit
+  ];
+
```

consider setting `strictDeps = true;` to ensure that you've added your deps to the correct place (native vs non-native inputs). 
```suggestion

  strictDeps = true;

```